src/node_options.cc

```diff
@@ -59,6 +59,15 @@ void DebugOptions::CheckOptions(std::vector<std::string>* errors) {
   }
 }
 
+PerProcessOptions::PerProcessOptions() {
+  // Default max worker thread count is one less
+  // than 4 times the total number of CPUs available.
+  CPUInfo cpu_info;
+  max_worker_thread_count = (cpu_info ?
+      cpu_info.count() * kMaxWorkerThreadMultiplier :
+      kMaxWorkerThreadMultiplier) - 1;
+}
```

Comment:
I’d suggest putting this computation into the class definition for consistency with the other options (you can still put the code in a helper function if you like)

Reply:
Ok. I tend to prefer keeping calculated init values in the constructor but can move it out to the definition.